src/isomorphic/hooks/ReactComponentTreeDevtool.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright 2016-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ReactComponentTreeDevtool
+ */
+'use strict';
+
+// TODO remove this proxy when RN/www gets updated
```

Comment:
yeah, I added backward compatibility support for `ReactDebugTool` in https://github.com/facebook/react/pull/7381/commits/bba0d992d8f4ecf9cf6677817a1218e7f48a8a77. Not sure if we should also do it for `ReactDOMDebugTool` tho :)